client/app/components/queries/editor-components/databricks/DatabricksSchemaBrowser.less

```diff
@@ -58,6 +62,5 @@
 }
 
 .databricks-schema-browser-db-dropdown {
-  width: auto !important;
-  max-width: 50vw;
+  width: 50vw !important;
```

Comment:
Since we have windowing now, the auto width doesn't work with it. Also the prop `dropdownMatchSelectWidth` automatically disables the windowing.